.github/workflows/CI-docker-sanity.yml

```diff
@@ -0,0 +1,62 @@
+name: Build and Test k-NN
+on:
+  push:
+    branches:
```

Comment:
This action will still fail if latest ODFE docker version is unavailable. Is there a way we can check to see if the docker image is available before running the test?

Reply:
Good catch. For now updated to dummy branch. We will have to fix it to run on demand.
created issue to unblock the release https://github.com/opendistro-for-elasticsearch/k-NN/issues/121

Reply:
Instead of adding it now with dummy branch, I think it might be better off to leave it out of this PR.

Reply:
makes sense. Removed